client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss

```diff
@@ -14,7 +17,10 @@
 	position: fixed;
 	tab-size: 4;
 	top: 0;
-	transition: color 0.5s ease-in-out 1s, background-color 0.5s ease-in-out 1s;
+	transition: color $acquire-intent-vertical-transition-duration
+			$acquire-intent-vertical-transition-algorithm,
+		background-color $acquire-intent-vertical-transition-duration
+			$acquire-intent-vertical-transition-algorithm;
```

Comment:
```suggestion
	transition-property: color, background-color;
	transition-duration: $acquire-intent-vertical-transition-duration;
	transition-timing-function: $acquire-intent-vertical-transition-algorithm;
```

Reply:
What's the motivation for this?

Reply:
Use the variables once, easier maintenance and subjectively easier to read, esp with such verbose names.

Reply:
You have to match the list lengths or defaults will be filled in, meaning:

```css
* {
	transition-property: color, background-color;
	transition-duration: 100ms;
	transition-timing-function: ease-in-out;
}
/* is equal to */
* {
	transition: color 100ms ease-in-out, background-color 0s ease;
}
```